api.c

```diff
@@ -127,6 +127,10 @@ static int api_handle_req_add_hostfwd(Slirp * slirp, int fd, struct api_ctx *ctx
 		free(fwd);
 		goto finish;
 	}
+
+	if (guest_addr_s == NULL || guest_addr_s[0] == '\0') {
+		guest_addr_s = "10.0.2.100";
```

Comment:
I'm not huge fan of this, because the guest addr is unknown when slirp4netns was not launched with `-c`

Reply:
Please add `when guest_addr is empty, guest_addr will be set to the default address that corresponds to --configure` to the man page. Then LGTM

Reply:
Do you want to define default `host_addr` as well? (0.0.0.0?)

Reply:
yes good idea, so they behave in a similar way

Reply:
The address needs to be now configurable: https://github.com/rootless-containers/slirp4netns/pull/62

Probably `do_slirp` should pass `slirp_config->recommended_vguest` to `api_ctx`